maintenance/health/servicehealthcheck/readable_health_handler.go

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2019 by PACE Telematics GmbH. All rights reserved.
+// Created at 2019/12/05 by Charlotte Pröller
+
+package servicehealthcheck
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+)
+
+// saves length of the longest name for the column width in the table. 20 characters width is the default
+var longestCheckName = 20
```

Comment:
If this a package internal value and we have no way of configuring (did I miss something) then we can make this a `const`, can't we?

Reply:
The Value is changed for each new registered health check (in `RegisterHealthCheck` and `RegisterOptionalHealthCheck`)

Reply:
Sorry, I must have misread the comment. It's explained right there :D